<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QMainWindow" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paaikkuna</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3" colspan="2">
     <widget class="QPushButton" name="muunna_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Muunna</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="tulos_label">
      <property name="text">
       <string>Tulos:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="4">
     <widget class="QLabel" name="tulos_maara_label">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="lopeta_button">
      <property name="text">
       <string>Lopeta</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QComboBox" name="muutettavat_dropdown">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="4">
     <widget class="QLabel" name="haku_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QPushButton" name="valitse_kohde">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Valitse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="maara_label">
      <property name="text">
       <string>Maara:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="2">
     <widget class="QPushButton" name="valitse_lahto">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Valitse</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="kohde_label">
      <property name="text">
       <string>Kohdeyksikko</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="muutettava_yksikko">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QComboBox" name="kohdeyksikot_dropdown">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="lahto_label">
      <property name="text">
       <string>Lahtoyksikko</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="tiedoston_nimi_label">
      <property name="text">
       <string>Tiedoston nimi:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="tiedosto_kentta"/>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QPushButton" name="lue_button">
      <property name="text">
       <string>Lue tiedosto</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuYksikko_muunnin">
    <property name="title">
     <string>Yksikko muunnin</string>
    </property>
   </widget>
   <addaction name="menuYksikko_muunnin"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lopeta_button</sender>
   <signal>clicked()</signal>
   <receiver>Paaikkuna</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
